libpf/interpretertype.go

```diff
@@ -55,13 +55,13 @@ var interpreterTypeToString = map[InterpreterType]string{
 	UnknownInterp: "unknown",
 	PHP:           "php",
 	PHPJIT:        "phpjit",
```

Comment:
Currently there is no `phpjit` semantic convention, just `php`, so we should also introduce it (unless we think there is no value in the distinction between the two). To me, it seems useful to be keep the distinction.

Reply:
Hmm. A lot of language runtimes have multiple execution tiers like "interpreted", "jit" or in the case of HotSpot even multiple tiers of JITing (C1 and C2). Perhaps instead of duplicating each frame kind, we should consider a separate attribute for that?

Reply:
We discussed this and decided to wait until @florianl comes back as there's a decision to be made also for error frames and I lack context.

Reply:
We also discussed this today and think that it makes sense to introduce a new attribute for per-frame interpreter-specific metadata which may include:

1. Error frame designation and error code
2. Inline status
3. JIT optimization / compiler level

We can discuss the specifics in the SIG.

As far as this PR is concerned, I think we can keep the frame type attribute and its string encoding, but this PR needs to be amended to be compliant with semantic conventions by:

1. Sending PHPJIT frame types as "php"
2. Stripping error meta information (e.g. send "jvm" instead of "jvm-error") and abort frames.

@florianl What do you think?